perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -57,7 +57,6 @@ you already have, and the countres set so that they shouldn't in future.
 
 sub setup_states {
     FixMyStreet::DB::Result::Problem->visible_states_add('unconfirmed');
-    FixMyStreet::DB::Result::Problem->visible_states_remove('investigating');
```

Comment:
It feels like this should have tests.

Reply:
Yes it should because then I'd have found the bug that Zurich just spotted :)